docs/intermediate_representation/tensors.md

```diff
@@ -0,0 +1,154 @@
+# Tensor representation in the IR
+
+The ONNX IR offers the :class:`TensorProtocol` interface for usings different data structures as backing data for tensors. Besides the traditional ``onnx.TensorProto``, you can also use ``np.ndarray``, ``torch.Tensor``, and virtually anything else to represent tensors in the graph. This allows for  They can be accessed and serialized via the same `TensorProtocol` interface.
```

Comment:
change "They can be" to "them to be"?

Reply:
Thanks for the catch. That was incomplete 😅